project/Build.scala

```diff
@@ -74,6 +76,29 @@ object Build {
   val publishedDottyVersion = referenceVersion
   val sbtDottyVersion = "0.5.5"
 
+  /** Version against which we check binary compatibility.
+   *
+   *  This must be the latest published release in the same versioning line.
+   *  For example, if the next version is going to be 3.1.4, then this must be
+   *  set to 3.1.3. If it is going to be 3.1.0, it must be set to the latest
+   *  3.0.x release.
+   */
```

Comment:
Can't we derive this from the version number?

Reply:
No. When the base version is 3.1.0, how do you know what's the previous version? It could be any 3.0.x for any value of *x*. You don't know without going through all the published version to determine which one is the latest 3.0.x.

Reply:
We could get away with that. That's not the established practice, though. And it means that you lose the ability to reasonably check experimental stuff that you introduce in the meantime, which you'd like not to *accidentally* break, at least.

IMO that's not a good trade off, but I can change if you prefer that it be automatically determined.

Reply:
I think it's fine as is if it's established practice, but we need to add the need to update these variables to the release checklist /cc @anatoliykmetyuk